Login Redirection to Dashboards

Irisvella opened the pull request:
#53 #35
- "remember me" function works
- Routes users to correct relevant dashboards when loggin in (admin and managers)
- Creating the backend APIs for such features.
- Fixed user model to use custom one instead of default django user model

Irisvella committed:
Login Redirection to Dashboards

#53 #35
- "remember me" function works
- Routes users to correct relevant dashboards when loggin in (admin and managers)
- Creating the backend APIs for such features.

Irisvella committed:
Merge branch 'main' into login-routing

Irisvella committed:
Added a missing comma

Fatoumatabintabarry reviewed (approved):
Everything works perfectly

Irisvella committed:
Rerrange App Routes + Add comments to current files

- Reorganized all app routes intro a defined structure for the future
- Added comments to files to explain what they do
- Some setup work on #15

inas-fawzi reviewed (approved):
seems good.
